### PR TITLE
Remove further shims from ReactDOM FBEntry

### PR DESCRIPTION
**Pull request by trueadm**
This removes further forward module shims from the ReactDOM FB Entry point, pending downstream changes on FB www.

Note: I pushed changes from an existing branch (`move-out-react-dom-shims`), so all commits can be historically tracked in relation to the changes to ReactDOM shims.

### CODE REVIEW

File: src/fb/ReactDOMFiberFBEntry.js [hunk 1]
```diff
@@ -14,20 +14,29 @@ var ReactDOMFiber = require('ReactDOMFiberEntry');
...
     ReactErrorUtils: require('ReactErrorUtils'),
+    // used by react-fb/ReactDOM on FB www
+    DOMProperty: require('DOMProperty'),
     ReactFiberErrorLogger: require('ReactFiberErrorLogger'),
-    ReactFiberTreeReflection: require('ReactFiberTreeReflection'),
-    ReactDOMComponentTree: require('ReactDOMComponentTree'),
+    // ReactInstanceMap are used to track instances
+    // for reactComponentExpect and ReactLayeredComponentMixin_DEPRECATED
     ReactInstanceMap: require('ReactInstanceMap'),
-    // This is used for ajaxify on www:
```
Comment (gaearon) [inline src/fb/ReactDOMFiberFBEntry.js]: This comment explains *why* we use it, can we keep it? It got lost with the move of this line.

File: scripts/rollup/shims/facebook-www/ReactErrorUtils.js [hunk 2]
```diff
@@ -1,19 +0,0 @@
-/**
- * Copyright 2013-present, Facebook, Inc.
- * All rights reserved.
- *
- * This source code is licensed under the BSD-style license found in the
- * LICENSE file in the root directory of this source tree. An additional grant
- * of patent rights can be found in the PATENTS file in the same directory.
- *
- * @providesModule ReactErrorUtils
```
Comment (gaearon) [inline scripts/rollup/shims/facebook-www/ReactErrorUtils.js]: We need this one.

Comment (trueadm) [inline scripts/rollup/shims/facebook-www/ReactErrorUtils.js]: We don't use the shim on www, we reference it through the secret internal object directly.

**Comment by gaearon** [comment]
Stale + we didn't land that www diff. Let's try again some time.

**Pull request closed without merging**